client/reducers/planning.js

```diff
@@ -43,6 +43,10 @@ const planning = (state={}, action) => {
             }
         case 'CLOSE_PLANNING_EDITOR':
             return { ...state, editorOpened: false, currentPlanningId: null }
+        case 'OPEN_ADVANCED_SEARCH':
```

Comment:
This fails because the ESLint code complexity doesn't like `switch ... case`.
You can either
. move the `RECEIVE_PLANNINGS` case implementation in a function
. increment the tolerated code complexity index
. convert the `switch case` with `if` statements